apps/src/accounts/TurnOffAiDiff.tsx

```diff
@@ -16,36 +16,28 @@ import i18n from '@cdo/locale';
 import moduleStyles from './accountSettings.module.scss';
 
 const TurnOffAiDiff: React.FC = () => {
-  const reduxState = useAppSelector(
+  const aiDifferentiationEnabled = useAppSelector(
     state => state.currentUser.aiDifferentiationEnabled
   );
 
   const currentUserId = useAppSelector(state => state.currentUser.userId);
 
-  const startingState = () => {
-    if (reduxState === null) {
-      new UserPreferences().setAiDifferentiationEnabled(true);
-      return true;
-    } else {
-      return reduxState;
-    }
-  };
-
-  const [hasAIDiffAccess, setHasAIDiffAccess] = React.useState(startingState);
-
   const handleToggle = () => {
     analyticsReporter.sendEvent(EVENTS.AI_DIFF_CHAT_TOGGLED, {
       'user id': currentUserId,
-      state: !hasAIDiffAccess ? 'on' : 'off',
+      state: aiDifferentiationEnabled ? 'on' : 'off',
```

Comment:
At this point `aiDifferentiationEnabled` has the old value so I think we need to flip the value here.

```suggestion
      state: !aiDifferentiationEnabled ? 'on' : 'off',
```